snscrape/modules/twitter.py

```diff
@@ -1081,7 +1081,12 @@ def _make_tweet(self, tweet, user, retweetedTweet = None, quotedTweet = None, ca
 						card.url = u.url
 						break
 				else:
-					_logger.warning(f'Could not translate t.co card URL on tweet {tweetId}')
+					try:
+						u = self._head(card.url)
+						assert u.status_code >= 300 and u.status_code < 400
+						card.url = u.headers["location"]
+					except:
```

Comment:
Bare `except`s aren't a great idea (they catch everything, including Ctrl-C), do you just want to catch AssertionError? If so, why not just use an if statement?

Reply:
> do you just want to catch AssertionError?

AssertionError or any exception thrown from requests (I think it's better to continue with the t.co URL and log the warning than to crash)

Reply:
In that case I'd either specify the exceptions or catch Exception (rather than the default BaseException, which catches EVERYTHING, including Ctrl-C):

```
except (AssertionError, WhateverElseYouWantToHandle, ...) as e:
    ...
```

or:

```
except Exception:
    ...
```

I'd recommend the former so you don't stifle valid errors.